src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
@dgoodwin I am relatively certain that this logic is identical to the previous logic -- 

The only diff between the `VersionTagger` implementation of `_tag_release` and the `ForceVersionTagger` implementation is that the `ForceVersionTagger` is passing `force` in as `True`. If there exists a `self._use_version`, this is identically the case when the `ForceVersionTagger` would have been in use anyway, so as far as I can tell, if we change the check on the `VersionTagger` to simply look for the existence of the `_use_version` field we have the same behavior.